generated/nirfmxwlan/nirfmxwlan.proto

```diff
@@ -1012,6 +1019,12 @@ enum NiRFmxWLANInt32AttributeValues {
   NIRFMXWLAN_INT32_OFDMMODACC_NOISE_COMPENSATION_INPUT_POWER_CHECK_ENABLED_TRUE = 1;
   NIRFMXWLAN_INT32_OFDMMODACC_OPTIMIZE_DYNAMIC_RANGE_FOR_EVM_ENABLED_FALSE = 0;
   NIRFMXWLAN_INT32_OFDMMODACC_OPTIMIZE_DYNAMIC_RANGE_FOR_EVM_ENABLED_TRUE = 1;
+  NIRFMXWLAN_INT32_OFDMMODACC_PHASE_ROTATION_COEFFICIENT1_PLUS_ONE = 0;
```

Comment:
Are these the names we want? Do they match the underscores in the header?

Or like `NIRFMXWLAN_INT32_OFDMMODACC_PHASE_ROTATION_COEFFICIENT_1_PLUS_ONE` ?

Reply:
Yeah, it looks like the header has an `_` between `COEFFICIENT` and `1`. That actually needs to be fixed in `scrapigen`. I'm not sure if you've encountered the code that tokenizes stuff (and overrides the defaults). I'm not totally fresh on how it works, but I can help dig into it if needed (or Ryan and David could).

You can get this in a separate change if you want, but can you add a task on it?

Reply:
I've fixed it in `enums.py` here (and therefore the generated `.proto`), but opened an issue in scrapigen. Thanks for catching this!